src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,269 @@
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-base-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Hover
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-hover-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Focus
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-focus-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Selected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-selected-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Disabled
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-disabled-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+        </table>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      
+      <div class="sky-text-block">
+        Keep titles and descriptions short and concise.  Limit descriptions to a single sentence.
+      </div>
+      
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      
+      <sky-docs-design-guideline>
+        <p>
+          Selection boxes are arranged in a centered container based on their number and the container’s width. Don’t use different layouts with selection boxes.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do center selection boxes and match their heights."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-layout-1-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use custom flex layouts to arrange selection boxes."
```

Comment:
We mention flex layouts, but consumers don't need a flex layout to make selection boxes left-aligned given our first-class property for that. Is the flex layouts piece here the correct thing to be calling out? Also, do we need to say anything about different heights here since the image demonstrates selection boxes that are different heights?